server/schedulers/grant_leader.go

```diff
@@ -252,7 +252,7 @@ func (s *grantLeaderScheduler) Schedule(cluster schedule.Cluster, dryRun bool) (
 			continue
 		}
 		op.Counters = append(op.Counters, schedulerCounter.WithLabelValues(s.GetName(), "new-operator"))
-		op.SetPriorityLevel(core.HighPriority)
+		op.SetPriorityLevel(core.High)
```

Comment:
`grant leader` and `grant hot leader` are used in test, so the priority should be same with the shuffle.

Reply:
IMO, grant leader can also be used in the production environment.